Index.php

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Painting Collection</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="css/normalise.css" type="text/css" rel="stylesheet">
+        <link href="css/style.css" type="text/css" rel="stylesheet">
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css2?family=Titillium+Web:ital,wght@0,400;0,600;0,700;1,400&display=swap" rel="stylesheet">
+    </head>
+    <body class="index-body">
+        <div class="index-title">
+            <h1>Welcome Inside Your Painting Collection</h1>
+        </div>
+        <nav class="navbar container">
+            <div>
+                <a href="gallery.php" class="">Explore Your Gallery</a>
+            </div>
+            <div>
+                <a href="form.php" class="">Add To You Collection</a>
```

Comment:
This is part of story2, please delete these files from story1 branch and put them in story2 branch.